docs/alternator/compatibility.md

```diff
@@ -16,7 +16,16 @@ provision that - Scylla requires you to provision your cluster. You need
 to reason about the number and size of your nodes - not the throughput.
 
 When creating a table, the BillingMode and ProvisionedThroughput options
-are ignored by Scylla.
+are ignored by Scylla. Tables default to a Billing mode of `PAY_PER_REQUEST`
+and `DescribeTable` API calls will return a value of 0 for the RCUs, WCUs
+and NumberOfDecreasesToday within the response.
+
+Another notable difference between DynamoDB and Scylla comes down to the
+underlying hashing algorithm. While DynamoDB uses a proprietary hashing
```

Comment:
The paragraph introducing this difference should mention **hash key** (a.k.a. partition keys), otherwise it's not clear what this "hashing" mentioned in this paragraph refers to (a reader might guess, but I think it won't be obvious to everyone).